editor/components/inserter/menu.js

```diff
@@ -176,6 +184,18 @@ export class InserterMenu extends Component {
 				/>
 
 				<div className="editor-inserter__results">
+					{ !! childItems.length &&
+						<div className="editor-inserter__child-blocks">
+							<div className="editor-inserter__parent-block-header">
+								<div className="editor-inserter__parent-block-icon">
+									<BlockIcon icon={ rootBlockIcon } />
```

Comment:
BlockIcon should be rendered only when the icon is set

Reply:
Normally we never do this check because BlockIcon handles it but in this case, we can do it because we have wrapping div for the icon.